heat/core/dndarray.py

```diff
@@ -566,6 +566,54 @@ def __cast(self, cast_function) -> Union[float, int]:
 
         raise TypeError("only size-1 arrays can be converted to Python scalars")
 
+    def collect_(self, target_rank: Optional[int] = 0) -> None:
+        """
+        A method collecting a distributed tensor to one rank, chosen by the target_rnk variable, and zero by default.
+        It is a specific case of the redistribute_ method.
```

Comment:
```suggestion
        It is a specific case of the ``redistribute_`` method.
```

Reply:
The double quotes turn `redistribute_` into a link to the function in the documentation.